code/html/index.html

```diff
@@ -620,6 +620,34 @@ <h2>Device administration and security settings</h2>
                                     <div class="pure-u-1 pure-u-lg-3-4 hint">Request password when starting telnet session</div>
                                 </div>
 
+                                <div class="pure-g">
+                                    <label class="pure-u-1 pure-u-lg-1-4">Heartbeat mode</label>
+                                    <select class="pure-u-1 pure-u-lg-1-4" name="hbMode" tabindex="15" >
+                                        <option value="0">HEARTBEAT_NONE</option>
+                                        <option value="1">HEARTBEAT_ONCE</option>
+                                        <option value="2">HEARTBEAT_REPEAT</option>
+                                        <option value="3">HEARTBEAT_REPEAT_STATUS</option>
+                                    </select>
+                                    <div class="pure-u-0 pure-u-lg-1-2"></div>
+                                    <div class="pure-u-0 pure-u-lg-1-4"></div>
+                                    <div class="pure-u-1 pure-u-lg-3-4 hint">
+                                        <strong>HEARTBEAT_NONE:</strong> no heartbeat at all, never.<br />
+                                        <strong>HEARTBEAT_ONCE:</strong> only on device startup.<br />
+                                        <strong>HEARTBEAT_REPEAT:</strong> on device startup and after HEARTBEAT_INTERVAL.<br />
+                                        <strong>HEARTBEAT_REPEAT_STATUS:</strong> on device startup full heartbeat information and after HEARTBEAT_INTERVAL only STATUS report.
+                                    </div>
+                                </div>
+
+                                <div class="pure-g">
+                                    <label class="pure-u-1 pure-u-lg-1-4">Heartbeat interval</label>
+                                    <input name="hbInterval" class="pure-u-1 pure-u-lg-1-4" type="text" tabindex="16" />
```

Comment:
This should be defined as a number, and probably in seconds, not millis.

Reply:
Yes, I was thinking about sec as well. Will do a fix. How would you like to store that setting in flash as sec or millis?